sched/mqueue/mq_initialize.c

```diff
@@ -49,12 +49,11 @@
 
 struct msgpool_s
 {
-#ifndef CONFIG_DISABLE_MQUEUE
+#ifdef CONFIG_DISABLE_MQUEUE_SYSV
```

Comment:
but, the user could disable both message queue api

Reply:
ifneq ($(CONFIG_DISABLE_MQUEUE)$(CONFIG_DISABLE_MQUEUE_SYSV),yy)
  CSRCS += mq_initialize.c mq_waitirq.c mq_recover.c
endif

when both disabled, mq_initialize.c will not compile